testing/src/test/java/io/stargate/it/http/RestApiTest.java

```diff
@@ -1224,24 +1253,22 @@ private void createTableWithClustering(String tableName) throws IOException {
     assertThat(successResponse.getSuccess()).isTrue();
   }
 
-  private void createTableWithMixedClustering(String tableName) throws IOException {
+  private void createTestTable(
```

Comment:
I have added this because I somehow did now want to copy all this boilerplate about ColumnDefinition.
Please let me know if this concept makes sense, if so then we can use it more widely in this test and document.
If not I will remove it and write a unit test for coverters / varint.